benchmarks/setup/default/zeebe-values.yaml

```diff
@@ -26,6 +31,7 @@ JavaOpts: |
   -XX:+PrintFlagsFinal
   -Xmx4g
   -Xms4g
+  -XX:+ExitOnOutOfMemoryError
```

Comment:
Does it work with the heapDump option? Because without the heap dump we are not able to root cause our memory issues.

Reply:
Yes, you will get a heap dump. The downside here is if you get in a crash loop backoff, the heap dump uses the PID, and the PID is always the same, so it fails to dump the second time, but this is already the case as is now anyway. The one downside is that `ExitOnOutOfMemoryError` does not shutdown gracefully =/